### ISSUE TITLE
SES domain identities are not verifying 

### ISSUE THREAD

**Issue by AlexClineBB** [Netflix/security_monkey#272]
Hello,

My AWS account has two SES domain identities and both are showing up in SM as not verified:

![screen shot 2015-12-07 at 1 06 29 pm](https://cloud.githubusercontent.com/assets/5075173/11635238/6d256ec4-9ce3-11e5-9046-d9b420c22ba4.png)

When I look in the AWS web console, they both say they're verified and when I use the API, I get confirmation that they're verified.

```
aws --region us-east-1 ses get-identity-verification-attributes --identities not.real.domain.tld
{
    "VerificationAttributes": {
        "not.real.domain.tld": {
            "VerificationToken": "+B...snip...0=", 
            "VerificationStatus": "Success"
        }
    }
}
```

Might SM not be picking up the fact that the domains were unverified when it first saw them and are now verified? Is there a test I should do to see what SM sees from the AWS API?

Thanks.

**Comment by monkeysecurity** [comment]
Interesting.

I would stop the scheduler and run the SES changewatcher manually on that account to see what it says:

    sudo supervisorctl
    stop securitymonkeyscheduler
    status
    CTRL-D

    export SECURITY_MONKEY_SETTINGS=whereverthatis
    python manage.py find_changes -m ses -a YOURACCOUNT

**Comment by AlexClineBB** [comment]
@monkeysecurity, here's the output from running that:

```
SECURITY_MONKEY_SETTINGS=/root/.secmonkey/config-deploy.tmp.py python manage.py find_changes -m ses -a BrandingBrand
Checking ses/BrandingBrand/us-west-2
Found 0 SES Identities. 0 are verified.
Checking ses/BrandingBrand/us-east-1
Found 6 SES Identities. 4 are verified.
Checking ses/BrandingBrand/eu-west-1
Found 0 SES Identities. 0 are verified.
Asked to audit 0 Objects
methods: [<bound method SESAuditor.check_verified of <security_monkey.auditors.ses.SESAuditor object at 0x7f01956e9410>>]


Saving Issues.
Creating/Assigning Auditor Settings in account [u'BrandingBrand'] and tech ses
Done Creating/Assigning Auditor Settings in account [u'BrandingBrand'] and tech ses
0 deleted SES Identities in [u'BrandingBrand']
0 created SES Identities in [u'BrandingBrand']
0 changed SES Identities in [u'BrandingBrand']
```

**Comment by monkeysecurity** [comment]
Ohh interesting. - Upgrading to bug.

**Comment by monkeysecurity** [comment]
Fixed by #335.
